server/metadata/tenant.go

```diff
@@ -135,6 +137,13 @@ func newTenantManager(kvStore kv.KeyValueStore, searchStore search.Store, mdName
 	}
 }
 
+func (m *TenantManager) EnsureDefaultNamespace(txMgr *transaction.Manager) error {
```

Comment:
What type of issues are you seeing?

Reply:
Tenant is required in quota middleware earlier than session manager is called.
Also creating it in session manager looks a bit hacky.

Reply:
But why do you think it is hacky? It is the default namespace provided by the cluster if there is no namespace. It is only required in the middleware in the first request? But default namespace, we can create during startup if it is missing in the cluster?